db/schema.rb

```diff
@@ -1,62 +1,62 @@
-    t.string "name"
-    t.string "photo"
-    t.text "bio"
-    t.integer "posts_counter"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-  end
-
-  add_foreign_key "comments", "posts", column: "post_id_id"
-  add_foreign_key "comments", "users", column: "user_id_id"
-  add_foreign_key "likes", "posts", column: "post_id_id"
-  add_foreign_key "likes", "users", column: "user_id_id"
-  add_foreign_key "posts", "users", column: "author_id_id"
-end
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 2023_10_27_125416) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "adminpack"
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.bigint "user_id_id"
+    t.bigint "post_id_id"
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id_id"], name: "index_comments_on_post_id_id"
+    t.index ["user_id_id"], name: "index_comments_on_user_id_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "user_id_id"
+    t.bigint "post_id_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id_id"], name: "index_likes_on_post_id_id"
+    t.index ["user_id_id"], name: "index_likes_on_user_id_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "author_id_id"
+    t.string "title"
+    t.text "text"
+    t.integer "comments_counter", default: 0
+    t.integer "likes_counter", default: 0
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id_id"], name: "index_posts_on_author_id_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.text "bio"
+    t.integer "posts_counter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_foreign_key "comments", "posts", column: "post_id_id"
+  add_foreign_key "comments", "users", column: "user_id_id"
+  add_foreign_key "likes", "posts", column: "post_id_id"
+  add_foreign_key "likes", "users", column: "user_id_id"
+  add_foreign_key "posts", "users", column: "author_id_id"
```

Comment:
- [ ] Good job so far 👏✨ However, as the previous reviewer mentioned, kindly make sure that all tests are passing. Currently, most of the tests are failing ⚠Please make sure to run your tests before asking for another review.
_**Hint: In your `schema.rb` you have `author_id_id` instead of `author_id` 🐛 Kindly ensure that you have correct references**_


![image](https://github.com/waelelsafty07/Blog/assets/106119105/2083de40-0b4f-41d2-aa39-0422bdaffc04)
![image](https://github.com/waelelsafty07/Blog/assets/106119105/f9884fd9-8b54-4728-b8a0-5ce3315657d6)